docs/crocks/Either.md

```diff
@@ -0,0 +1,89 @@
+# Either
+
+`Either a b`
+
+Crock which --
+
+The "`Either`" crock creates disjunction with a linear data flow.
+
+```js
+--
+```
+
+`Either` exposes these constructors and instances:
+
+| Constructor | Instance |
+|:---|:---|
+| [`Left`](#left), [`Right`](#right), [`of`](#of) | [`inspect`](#inspect), [`value`](#value), [`type`](#type), [`equals`](#equals), [`map`](#map), [`ap`](#ap), [`of`](#of), [`chain`](#chain), [`sequence`](#sequence), [`traverse`](#traverse) |
+
+## Constructors
+
+### Left
+
+`Either m => a -> m a _`
+
+### Right
+
+`Either m => a -> m _ a`
+
+### of
+
+`Either m => a -> m _ a`
+
+## Instances
+
+### inspect
+
+`() => String`
+
+### either
+
+`Either m => m c a ~> (c -> b) -> (a -> b) -> b`
+
+### value
+
+`Either m => m c a ~> () => c | a`
+
+### type
+
+`() -> String`
+
+### swap
+
+`Either m => m a b ~> m a b`
```

Comment:
This should swap the types, as it negates the or.

so something like:
`Either m => m a b ~> m b a`